spec/requests/topics_controller_spec.rb

```diff
@@ -1232,12 +1243,12 @@ def topic_user_post_timings_count(user, topic)
           topic.reload
           expect(topic.title).to eq('This is a new title for the topic')
 
-          expect(Jobs::EmitWebHookEvent.jobs.length).to eq(2)
+          expect(Jobs::EmitWebHookEvent.jobs.length).to eq(1)
```

Comment:
Not related but let us change this to use the sidekiq helpers too :)

Reply:
I'm unable to use the new helper method in this case since it's [testing nested hash arguments](https://github.com/discourse/discourse/pull/10590/files/8e0cd8cd2882eac08a59f5ba409b5015ec680e84#diff-0fae1daa2e2270483f864ea4a3e859f7R1250-R1251).